packages/react/src/checkout/draftOrders/hooks/useDraftOrder.ts

```diff
@@ -0,0 +1,239 @@
+   *
+   * @returns Promise that will resolve when the call to the endpoint finishes.
+   */
+  const removeDraftOrderItem = useCallback(
+    (
+      itemId: DraftOrderItem['id'],
+      config: Config | undefined = fetchConfig,
+      draftOrderId: DraftOrder['id'] = draftOrderIdHookParameter,
+    ) => {
+      if (!draftOrderId) {
+        return Promise.reject(new Error('No draft order id was specified.'));
+      }
+
+      return removeItem(draftOrderId, itemId, config);
+    },
+    [fetchConfig, draftOrderIdHookParameter, removeItem],
+  );
+
+  /**
+   * Update the draft order. You can override the draft order id to update draft order by using
+   * the optional `draftOrderId` parameter. However, the output from
+   * the hook will respect the order id passed to it and not the override.
+   *
+   * @param data - Required the draft order data from the draft order id.
+   * @param config - Custom configurations to send to the client instance (axios). If undefined, the `fetchConfig` passed to the hook will be used instead.
+   * @param draftOrderId  - Overrides the draft order id from the hook. If undefined, the `draftOrderId` passed to the hook will be used instead. Note that the output of the hook will respect the `draftOrderId` parameter from the hook.
+   *
+   * @returns Promise that will resolve when the call to the endpoint finishes.
+   */
+  const updateDraftOrder = useCallback(
+    (
+      data: DraftOrderData,
+      config: Config | undefined = fetchConfig,
+      draftOrderId: DraftOrder['id'] = draftOrderIdHookParameter,
+    ) => {
+      if (!draftOrderId) {
+        return Promise.reject(new Error('No draft order id was specified.'));
+      }
+
+      return update(draftOrderId, data, config);
+    },
+    [fetchConfig, draftOrderIdHookParameter, update],
+  );
+
+  /**
+   * Update the draft order item. You can override the draft order id to update draft order item by using
+   * the optional `draftOrderId` parameter. However, the output from
+   * the hook will respect the order id passed to it and not the override.
+   *
+   * @param itemId - Required the draft order item id.
+   * @param dataItem - Required the draft order item.
+   * @param config - Custom configurations to send to the client instance (axios). If undefined, the `fetchConfig` passed to the hook will be used instead.
+   * @param draftOrderId  - Overrides the draft order id from the hook. If undefined, the `draftOrderId` passed to the hook will be used instead. Note that the output of the hook will respect the `draftOrderId` parameter from the hook.
+   *
+   * @returns Promise that will resolve when the call to the endpoint finishes.
+   */
+  const updateDraftOrderItem = useCallback(
+    (
+      itemId: DraftOrderItem['id'],
+      dataItem: PatchDraftOrderItemData,
```

Comment:
In this case I've kept the name because there are a lot of variables in these tests and it makes it simpler for anyone looking to understand that the updated data is for the item in a draft order.

Reply:
Done